packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -495,18 +496,25 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
   }
 
   // Returns the week numbers for each week in a month.  Week numbers are 1 - 52 (53) in a year
-  private _getWeekNumbersInMonth(weeksInMonth: number, firstDayOfWeek: DayOfWeek, navigatedDate: Date) {
+  private _getWeekNumbersInMonth(weeksInMonth: number, firstDayOfWeek: DayOfWeek, firstWeekOfYear: FirstWeekOfYear, navigatedDate: Date) {
```

Comment:
>_getWeekNumbersInMonth [](start = 10, length = 22)

Do we need to move getWeekNumbersInMonth() to DateMath also so we can call it from owa-calendar?

Reply:
good thought!